synapse/rest/admin/users.py

```diff
@@ -702,9 +702,10 @@ async def on_GET(self, request, user_id):
         if not self.is_mine(UserID.from_string(user_id)):
             raise SynapseError(400, "Can only lookup local users")
 
-        room_ids = await self.store.get_rooms_for_user(user_id)
-        if not room_ids:
-            raise NotFoundError("User not found")
+        u = await self.store.get_user_by_id(user_id)
+        if u is None:
```

Comment:
We should try to avoid single character variables.

```suggestion
        user = await self.store.get_user_by_id(user_id)
        if user is None:
```